bower.json

```diff
@@ -31,5 +29,8 @@
     "paper-item": "PolymerElements/paper-item#master",
     "test-fixture": "Polymer/test-fixture#master",
     "web-component-tester": "Polymer/web-component-tester#^2.2.0"
+  },
+  "resolutions": {
+    "polymer": "v0.8.0-rc.4"
   }
```

Comment:
How do I handle my dependencies having conflicting versions of a package? I don't think I should just let the tests fail.